book/src/help_bn.md

```diff
@@ -26,7 +26,7 @@ FLAGS:
                                                contention which degrades staking performance. Stakers should generally
                                                choose to avoid this flag since backfill sync is not required for
                                                staking.
-        --disable-deposit-contract-sync        Explictly disables syncing of deposit logs from the execution node. This
+        --disable-deposit-contract-sync        Explicitly disables syncing of deposit logs from the execution node. This
```

Comment:
The `help_bn.md` file is generated from the Lighthouse binary. So it is better if we correct for the typo from the source code here: 

https://github.com/sigp/lighthouse/blob/441fc1691b69f9edc4bbdc6665f3efab16265c9b/beacon_node/src/cli.rs#L1168

Could you revert this change, correct in the source code, and then run `make cli` from the root of Lighthouse depository? That will update the file directly

Reply:
> After running `make`, it appears that the mentioned typo was only present in that particular file.

The change on `help_bn.md` has to be reverted (i.e., make it as if no change), because updating `cli.rs` will update `help_bn.md` when the new binary is build.

Reply:
> @chong-he Reverted 👍

My bad. Once you update the `cli.rs` (which you have), then you can run `make cli`, which would update the `help_bn.md`, and you can commit it (so you still need the change on `help_bn.md`). Apologies for the confusion

Reply:
> @chong-he Reverted the reversion

Looks good, thank you